tests/integration/worker/test_submission_worker.py

```diff
@@ -203,6 +203,29 @@ def test_process_submission_message_when_challenge_phase_does_not_exist(self, mo
         mock_logger.assert_called_with("Challenge Phase {} does not exist".format(phase_pk))
 
 
+class ProcessAddChallengeMessageTestClass(BaseTestClass):
+    def test_process_add_challenge_message_successfully(self):
```

Comment:
@nsjcorps one minor change. Can you please rename this to `test_process_add_challenge_message`

Reply:
I have made the requested changes

Reply:
@Ram81